experimental/addons/chaosblade-operator/template.cue

```diff
@@ -0,0 +1,54 @@
+package main
+
+
+output: {
+        apiVersion: "core.oam.dev/v1beta1"
+        kind:       "Application"
+        spec: {
+                components: [
+                        {
+                                type: "k8s-objects"
+                                name: "blade-ns"
+                                properties: objects: [{
+                                        apiVersion: "v1"
+                                        kind:       "Namespace"
+                                        metadata: name: parameter.namespace
+                                }]
+                        },
+                        {
+                            name: "blade-helm"
+                            type: "helm"
+                            dependsOn: ["blade-ns"]
+                            properties: {
+                                    repoType: "helm"
+                                    url:      "https://oeular.github.io/chaosblade-operator/"
```

Comment:
Can we ref to an official address?

Reply:
The official helm chart repo is not ready. I have made an issue for them.